pkgs/development/tools/analysis/rizin/cutter.nix

```diff
@@ -8,7 +8,7 @@
 # buildInputs
 , graphviz
 , rizin
-, python3
+, python
```

Comment:
I'd hate to nitpick this PR too much, but was there a reason for changing this from `python3` to just `python`?

It'll presumably have to be changed back to `python3` when we update cutter to use Qt6/Pyside6

Reply:
Probably just for esthetics, but if there is a reason not to, we should leave it as python3.

Reply:
I generally still expect `python` to mean "Python 2", so imo this is a bad change